task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -20,6 +20,20 @@
 log = logging.getLogger(__name__)
 
 
+VOLUME_KEYS = set(['mode', 'container_path', 'host_path'])
```

Comment:
it seems like there is:

    In [3]: ['a', 'b'] == {'a': 1, 'b': 2}.keys()
    Out[3]: False

dict_keys supports equality comparison with sets but apparently not with lists